website/pages/en/network/curating.mdx

```diff
@@ -4,7 +4,7 @@ title: Curating
 
 Curators are critical to the Graph decentralized economy. They use their knowledge of the web3 ecosystem to assess and signal on the subgraphs that should be indexed by The Graph Network. Through the Explorer, curators are able to view network data to make signaling decisions. The Graph Network rewards curators who signal on good quality subgraphs with a share of the query fees that subgraphs generate. Curators are economically incentivized to signal early. These cues from curators are important for Indexers, who can then process or index the data from these signaled subgraphs.
 
-When signaling, curators can decide to signal on a specific version of the subgraph or to signal using auto-migrate. When signaling using auto-migrate, a curator’s shares will always be upgraded to the latest version published by the developer. If you decide to signal on a specific version instead, shares will always stay on this specific version.
+When signaling, curators can decide to signal on a specific version of the subgraph or to signal using auto-migrate. When signaling using auto-migrate, a Curator’s shares will always be updated to the latest version published by the developer. If you decide to signal on a specific version instead, shares will always stay on this specific version.
```

Comment:
Is "updating" shares the right term? Below (in question 3), we use the term "migrating":

> Migrating your curation shares to a new subgraph version...

Reply:
Good catch! Thank you!